WordPress/build.gradle

```diff
@@ -97,8 +97,6 @@ android {
         buildConfigField "boolean", "ENABLE_DEBUG_SETTINGS", "true"
         buildConfigField "boolean", "MY_SITE_IMPROVEMENTS", "false"
         buildConfigField "boolean", "SEEN_UNSEEN_WITH_COUNTER", "false"
-        buildConfigField "boolean", "CONTACT_INFO_BLOCK_AVAILABLE", "true"
```

Comment:
WDYT about removing the flag definition in the `vanilla` variant as well? `vanilla` is the production variant. Removing from there too will make clean up more complete.

Reply:
Good catch! I'd actually removed them in the first commits of this PR and then accidentally re-added them when resolving a conflict in https://github.com/wordpress-mobile/WordPress-Android/pull/15190/commits/f6e3664c94e4253c5faeed0406963b85563848d1 🤦‍♂️
I've re-removed them now 🙂